app/src/main/java/com/seafile/seadroid2/data/DataManager.java

```diff
@@ -183,7 +183,12 @@ public Account getAccount() {
     }
 
     private File getFileForReposCache() {
-        String filename = "repos-" + (account.server + account.email).hashCode() + ".dat";
```

Comment:
better to keep the suffix `.dat` unchanged I think, otherwise those already cached `.dat` files will lose control. Same thing to dirent caches.